### DISCUSSION TITLE
Grammar engineering meetings

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#144] [Announcements]
Dear @delph-in/participants , for those interested in regular grammar engineering discussions, please fill out the poll for the upcoming semester (or at least quarter):

https://www.when2meet.com/?22986264-K3GQY

(I would also like to remind people about the [2024 DELPH-IN poll](https://github.com/orgs/delph-in/discussions/143)!)

**Comment by olzama** [comment]
...I forgot to say, as usual, that the specific dates should be ignored and treated simply as days of the odd and even weeks of the upcoming months!

**Comment by arademaker** [comment]
I am done, but I don't yet have the semester scheduler, so what courses I will teach is unclear.

**Comment by olzama** [comment]
@arademaker , we can wait until your schedule is finalized, too. When should that become clearer?

**Reply by arademaker** [reply]
Do you mean not to fill the when2meet?

**Reply by olzama** [reply]
No it's fine, I just meant, if you know when you know your availability better, we can wait a bit before scheduling the meetings...

**Comment by emilymbender** [comment]
Not sure I can do every time, but I've filled it in. NB: alternate Tuesdays
are clear for me (with Jan 16 being one of the clear ones).

**Comment by olzama** [comment]
Any other @delph-in/participants would like to fill the when2meet before I schedule the meetings?

**Reply by olzama** [reply]
...to clarify, nobody (except me) is expected to join every time; people join when they can. But I think in the previous months, there was always someone else in addition to myself, I never ended up alone :). So, fill this out in case you would like to just join sometimes.

**Comment by olzama** [comment]
Alright, given that two Mondays next U.S. quarter are U.S. holidays, we will do Tuesdays (even weeks). I have scheduled meetings through March, and then we can change the schedule if needed for the Spring quarter (my own schedule is not going to change as we are on a semester system). I will create calendar events for the usual attendees; please let me know if you also want to be added, and I will also post reminders here.

**Comment by olzama** [comment]
Some of us will meet today (Tuesday the 16th) at 18:00 Madrid time! 

Olga Zamaraeva is inviting you to a scheduled Zoom meeting.

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will meet today (Tuesday the 30th) at 18:00 Madrid time!

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will meet today (Tuesday the 27th) at 18:00 Madrid time!

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

**Comment by fcbond** [comment]
I'm afraid I can't make it today.

On Tue, 27 Feb 2024 at 15:23, Olga Zamaraeva ***@***.***>
wrote:

> Dear @delph-in/participants
> <https://github.com/orgs/delph-in/teams/participants> ,
>
> Some of us will meet today (Tuesday the 27th) at 18:00 Madrid time!
>
> https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09
>
> Meeting ID: 980 1412 5772
> Passcode: 415301
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/144#discussioncomment-8605937>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRW63ZIIUMNPYM7BRQLYVXT4NAVCNFSM6AAAAABBRGPQYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMMBVHEZTO>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will meet Tuesday the 12th of March at 18:00 Madrid time!

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will meet Tuesday the 26th of March at 18:00 Madrid time!

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will meet Tuesday the 26th of March at 18:00 Madrid time!

https://zoom.us/j/98014125772?pwd=K3NYV0Vid21PbStNL1FGWkMvMGRhQT09

Meeting ID: 980 1412 5772
Passcode: 415301

P.S. I think in the U.S. it is now already another quarter, so if something changed for anyone and you can never make this time anymore, let me know. Otherwise we will leave everything as is, as it has been working pretty well.

**Comment by olzama** [comment]
Sorry, wrong meeting info!

Correct meeting info:

Topic: Grammar Engineering meeting 
Time: Apr 9, 2024 06:00 PM Madrid

Join Zoom Meeting
https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09

Meeting ID: 937 1764 9329
Passcode: 259291

**Comment by olzama** [comment]
Some of us will be meeting today 18:00 Madrid time!

Topic: Grammar Engineering meeting
Time: Apr 9, 2024 06:00 PM Madrid

Join Zoom Meeting
https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09

Meeting ID: 937 1764 9329
Passcode: 259291

**Comment by olzama** [comment]
Dear @delph-in/participants , 

Some of us will be meeting <s>today</s> **tomorrow** 18:00 Madrid time!

Topic: Grammar Engineering meeting
Time: May 7, 2024 06:00 PM Madrid

Join Zoom Meeting
https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09

Meeting ID: 937 1764 9329
Passcode: 259291

**Comment by olzama** [comment]
Sorry @delph-in/participants I of course meant the meeting is today on Tuesday May <s>6</s> **7.**

**Reply by arademaker** [reply]
Today is May 7.

**Reply by olzama** [reply]
I am a disaster :) Yes, today May 7.

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Some of us will be meeting today June 4 18:00 Madrid time!

https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09

Meeting ID: 937 1764 9329
Passcode: 259291

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Last meeting before this year's summit! :)

Some of us will be meeting today June 18 at 18:00 Madrid time!

https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09

Meeting ID: 937 1764 9329
Passcode: 259291

**Comment by danflick** [comment]
Sorry I can't join the discussion today.  See you in Olomouc!

 Dan

**Comment by emilymbender** [comment]
Likewise I'm not able to today... in Mexico for NAACL!

**Comment by fcbond** [comment]
And I'm stuck on a substitute train near Prague.

See you all soon!

On Tue, 18 Jun 2024 at 16:24, emilymbender ***@***.***> wrote:

> Likewise I'm not able to today... in Mexico for NAACL!
>
> On Tue, Jun 18, 2024, 8:22 AM Dan Flickinger ***@***.***>
> wrote:
>
> > Sorry I can't join the discussion today. See you in Olomouc!
> >
> > Dan
> >
> >
> > On Tue, Jun 18, 2024 at 5:06 AM Olga Zamaraeva ***@***.***>
> > wrote:
> >
> > > Dear @delph-in/participants
> > > <https://github.com/orgs/delph-in/teams/participants>
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/teams/participants*3E__;JQ!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJviJje6Y$>
>
> > ,
> > >
> > > Last meeting before this year's summit! :)
> > >
> > > Some of us will be meeting today June 18 at 18:00 Madrid time!
> > >
> > > https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09
> > <
> https://urldefense.com/v3/__https://zoom.us/j/93717649329?pwd=MkpUcEN4YTZCbnpWbGRKazAzSkpTZz09__;!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJ4mMJ2dI$>
>
> > >
> > > Meeting ID: 937 1764 9329
> > > Passcode: 259291
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/orgs/delph-in/discussions/144#discussioncomment-9804443>
>
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/144*discussioncomment-9804443*3E__;IyU!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJbVFi2rU$>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AG5PC4XD73O66YQNNFZYC6DZIABB3AVCNFSM6AAAAABBRGPQYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMBUGQ2DG>
>
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AG5PC4XD73O66YQNNFZYC6DZIABB3AVCNFSM6AAAAABBRGPQYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMBUGQ2DG*3E__;JQ!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJvKgsDHA$>
>
> > > .
> > > You are receiving this because you are on a team that was
> > mentioned.Message
> > > ID: ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/144*discussioncomment-9807056__;Iw!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJ4gHD4Hw$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6UFZDW7HBFBCHOP5MTZIA7BDAVCNFSM6AAAAABBRGPQYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMBXGA2TM__;!!K-Hz7m0Vt54!ijCDFfBAI8JQpOFQg3VnjTzCgidp1ES-VMRB4XabifFjfdw3B8dueEirDvnENZtJ74TJGmnkcGN5cZFJKZGcumA$>
>
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/144#discussioncomment-9807077>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRWHUM6VYQY3HJWUDNTZIA7IXAVCNFSM6AAAAABBRGPQYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMBXGA3TO>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by olzama** [comment]
OK if anyone *does* plan to join, I'd appreciate a quick message, because if not, then just canceling the meeting would help me with an errand tonight. However, I prefer to not cancel it if anyone does plan to show up.

**Comment by lmorgadodacosta** [comment]
Hi Olga,

I'll join! Let's talk grammar demos!

See you in about 1 hour.

Best,
L
